### PR TITLE
[Cache] Updates to Early Hints

### PR DESCRIPTION
**Pull request by dcpena**
Update to Early Hints that are covered in [PCX-2896](https://jira.cfops.it/browse/PCX-2896).

### CODE REVIEW

File: products/cache/src/content/about/early-hints.md [hunk 1]
```diff
@@ -21,20 +21,26 @@ After you sign up, you will be added to a list of beta users, and the feature wi
...
 Currently, only certain browser versions will take action to preload or preconnect on receiving Early Hints, such as Google Chrome M94 and higher. Instructions for running WebPageTest to experiment with compatible client browsers can be found in the [blog post](https://blog.cloudflare.com/early-hints/#testing-early-hints-with-web-page-test).
 
-<Aside type="note" >
-
-Early Hints responses are emitted prior to reaching the origin server or Worker. Note that if a page on your zone requires authenticated access, enabling Early Hints can serve a 103 response with previously cached Link headers to an unauthenticated visitor ahead of a 403 Forbidden response from your origin.
+Additionally, keep the following in mind:
 
-</Aside>
\ No newline at end of file
+- Early Hints responses may be emitted before reaching the origin server or Worker. If a page on your zone requires authenticated access, enabling Early Hints can serve a 103 response with previously cached Link headers to an unauthenticated visitor ahead of a 403 Forbidden response from your origin.
```
Comment (nayaabahsan) [inline products/cache/src/content/about/early-hints.md]: The second sentence here is hard to follow/understand. I don't have a suggestion because I don't fully understand what is being said, but I think it should be broken up.

**Review by nayaabahsan — APPROVED** [review]

File: products/cache/src/content/about/early-hints.md (hunk 1 above)
Comment (dcpena) [reply products/cache/src/content/about/early-hints.md]: Is this a little better? I tried shifting the subject

>If pages on your website require user authentication, unauthenticated visitors may receive a 103 response with previously cached Link headers before they receive a 403 Forbidden response from the origin when Early Hints is enabled.

Comment (nayaabahsan) [reply products/cache/src/content/about/early-hints.md]: It is better, but I think the sentence just needs to be broken up into separate sentences. I don't know if this makes technical sense, but maybe more like:

>When Early Hints is enabled and pages on your site require authentication, unauthenticated visitors may receive a 103 response. This 103 response would be emitted with previously cached Link headers before a 403 Forbidden response from your origin.

Comment (dcpena) [reply products/cache/src/content/about/early-hints.md]: @nayaabahsan omg, teamwork! thanks for finding a spot to break up the sentence. I'll make that change now
